streams/runners/storm/actions/src/main/java/com/hortonworks/streamline/streams/actions/storm/topology/StormTopologyActionsImpl.java

```diff
@@ -259,7 +266,7 @@ public void deploy(TopologyLayout topology, String mavenArtifacts, TopologyActio
     }
 
     @Override
-    public void testRun(TopologyLayout topology, String mavenArtifacts,
+    public void runTest(TopologyTestRunHistory testRunHistory, TopologyLayout topology, String mavenArtifacts,
```

Comment:
testRunHistory can be the second argument. All other methods take TopologyLayout as the first arg.

Reply:
Good point. Will address.